docusearch/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class Document(models.Model):
+    """ A model representing a document from an agency. """
+
+    text = models.TextField(
```

Comment:
I wonder if it would be useful/possible to only store the doc text in es. I think we might be able to use the [es text field](https://github.com/projectkrang/foia-hub/blob/basic_beginning/docusearch/templates/search/indexes/docusearch/document_text.txt) since we aren't adding anything to it.